test/basis.test.js

```diff
@@ -184,23 +184,6 @@ describe("Basis Test", () => {
             expect(falseIncludedSelectors).toHaveLength(0);
         });
 
-        test("Vendor prefixes still exists", () => {
```

Comment:
Why don't you write an additional test case for vendor prefixes and leave it for the NOCSS tests only. So we keep the vendors test like it is but don't use it for the NOCSS

Reply:
I'm not good in Jest to be honest :-) For me it's a bit complex now. 
I tried to split tests to no-css and with-css, but it make tests slower. 
If you can help me with this - I'll be happy  :-)

Reply:
Sorry for delay. Iam at vacation in the moment. Will have a look later.

Reply:
No stress.
I can use it anyway from my repo :-)
Have a good one!